article_store/articles/xml_parser.py

```diff
@@ -0,0 +1,67 @@
+from typing import Any, Tuple
+
+from django.conf import settings
+from rest_framework_xml.parsers import XMLParser
+import defusedxml.ElementTree as etree
+
+from xml.etree.ElementTree import register_namespace
+
+
+def register_namespaces(name_spaces: Tuple[Tuple[str, str]]) -> None:
+    """Register `n` number of xml namespaces
+
+    :param name_spaces: Tuple
+    :return:
+    """
+    for name_space in name_spaces:
+        register_namespace(*name_space)
```

Comment:
it's a pity that this is global rather than being on a `parser` object, but can't change a standard library :man_shrugging: